From e33702e5e5fe9fef6ec967961e2e5e1c2285ba36 Mon Sep 17 00:00:00 2001
From: gtrainavicius <gtrainavicius@users.noreply.github.com>
Date: Wed, 4 Dec 2024 11:18:14 +0200
Subject: [PATCH] =?UTF-8?q?Allow=20setting=20I=C2=B2C=20clock=20frequency?=
 =?UTF-8?q?=20via=20i2c=5Farm=5Fbaudrate=20dtparam=20when=20using=20pimidi?=
 =?UTF-8?q?=20overlay.?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This change removes the forced 1MHz clock frequency, so it can be overridden using `i2c_arm_baudrate`.
---
 arch/arm/boot/dts/overlays/pimidi-overlay.dts | 1 -
 1 file changed, 1 deletion(-)

--- a/arch/arm/boot/dts/overlays/pimidi-overlay.dts
+++ b/arch/arm/boot/dts/overlays/pimidi-overlay.dts
@@ -26,7 +26,6 @@
 		target = <&i2c_arm>;
 		__overlay__ {
 			status = "okay";
-			clock-frequency=<1000000>;
 
 			pimidi_ctrl: pimidi_ctrl@20 {
 				compatible = "blokaslabs,pimidi";
